src/controllers/routes.js

```diff
@@ -3,8 +3,14 @@ const express = require('express');
 const router = express.Router();
 
 const login = require('./login');
+const men = require('./men');
+const women = require('./women');
+
 
 router.get('/login', login.get);
 router.post('/loginuser', login.post);
+router.get('/men-fashion', men.get);
+router.get('/women-fashion', women.get);
```

Comment:
when i use this router /women-fashion , no data show . i think your database doesn't  contain test data for women-fashion . So add test data :smile:

Reply:
put the heroku database url in the ``` config.env ``` file, because i added the data into it :smile:

Reply:
sorry :smile:  i work as local database :+1: